src/templatizer.js

```diff
@@ -4,46 +4,41 @@ const argumentsTemplate = JSON.parse(fs.readFileSync('./src/template/args.json')
 const optionTemplate = require('./template/option.json');
 
 const templatizer = (context, config) => {
-    argumentsTemplate.definitions.arguments.properties = {};
+    argumentsTemplate.definitions.object.properties = {};
     let result = argumentsTemplate;
-    const customPrefix = config.prefixes.custom;
-    const nonFlagPrefix = config.prefixes.nonFlag;
     context.options.map((option) => {
         let optionSchema = Object.assign({}, optionTemplate);
         const argumentName = option.longName ? option.longName :
                 (option.shortName ? option.shortName : '');
         switch (argumentName) {
```

Comment:
A good candidate for a separate func.

Reply:
@xferra what do you mean? Move all inside map to a separate function? It's almost full templatizer function.